lib/vmdb/settings/validator.rb

```diff
@@ -104,7 +93,7 @@ def session(data)
             errors << [:timeout, "timeout, \"#{data.timeout}\", invalid. Should be numeric"]
           end
 
-          if data.timeout == 0
+          if data.timeout.kind_of?(Integer) && data.timeout.zero?
```

Comment:
I kind of preferred the original code...more concise and cleaner.  Why did you change?  If because of Rubocop, we can just ignore it.

Reply:
Yes, this was done to please the Rubocop.